tests/dogfood.rs

```diff
@@ -82,7 +82,16 @@ fn run_clippy_for_package(project: &str) -> bool {
 
     command.arg("--");
     command.arg("-Cdebuginfo=0"); // disable debuginfo to generate less data in the target dir
-    command.args(["-D", "clippy::all", "-D", "clippy::pedantic", "-D", "clippy::dbg_macro"]);
+    command.args([
+        "-D",
+        "clippy::all",
+        "-D",
+        "clippy::pedantic",
+        // "-D",                        // can't enable allow_attributes lint because of
+        // "clippy::allow_attributes",  // https://github.com/rust-lang/rust-clippy/issues/13358
```

Comment:
I would suggest making a separate issue for this and removing it from the code for now. I don't think whoever fixes that issue will remember to comment this out, and it *may* be worth discussing separately if this is something we want to do anyway

Reply:
Removed!